docker-compose.yml

```diff
@@ -14,10 +14,16 @@ services:
       SPLUNK__URL: ${SPLUNK_COLLECTOR_URL}
       SPLUNK__TOKEN: ${SPLUNK_HEC_TOKEN}
       TICKETSEARCHAPI__BASEURL: http://ticket-search:8080/
+      RABBITMQ__HOST: rabbitmq
+      RABBITMQ__PORT: 5672
+      RABBITMQ__USERNAME: guest
+      RABBITMQ__PASSWORD: guest
     ports:
       - "5000:8080"
     restart: always
-    depends_on: [splunk, rabbitmq]
+    depends_on: [splunk, rabbitmq, ticket-search]
+    networks:
+      - traffic-court-net
```

Comment:
I am not sure why this is required, we define the `default` network at the bottom, see https://docs.docker.com/compose/networking/#configure-the-default-network

```
Instead of (or as well as) specifying your own networks, you can also change the settings of the app-wide default network by defining an entry under networks named default
```

Reply:
that does not work. I do not know why. But after add that, it works.